openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -79,8 +79,11 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 }
 
 // GetExportLocations will get shareID's export locations
-func GetExportLocations(client *gophercloud.ServiceClient, id string) (r GetExportLocationsResult) {
+// client must have Microversion set; minimum supported microversion for Get Export Locations is 2.14
+func GetExportLocations(client *gophercloud.ServiceClient, micoversion string, id string) (r GetExportLocationsResult) {
```

Comment:
Microversion is already a part of the client `struct` that's why I think it's not necessary to another `microversion` parametr to the `GetExportLocations` function.

Reply:
Only improve the description of the function here